hasher-matcher-actioner/webapp/src/pages/settings/ActionRuleSettingsTab.tsx

```diff
@@ -44,20 +39,23 @@ export class ActionRule {
 
   must_not_have_labels: Label[];
 
-  action_id: string;
-
   classification_conditions: ClassificationCondition[];
 
+  action: string;
```

Comment:
Why the rename? `action_id` is not accurate? I think this adds more ambiguity personally.